### ISSUE TITLE
Python 3.4 is a dependency?

### ISSUE THREAD

**Issue by dustinmichels** [rnelsonchem/simpledbf#6]
Hi! I tried to conda install with `conda install -c rnelsonchem simpledbf `, and got the message:

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - python 3.6*
  - simpledbf -> python 3.4*
Use "conda info <package>" to see the dependencies for each package.
```

Does it really require 3.4? Or can that dependency be omitted from the package installation recipe?

Thanks!

**Comment by rnelsonchem** [comment]
Yeah. It probably works for all versions of Python 3. The code is just a
single file, so you could just download it from GitHub. Or else pip will
probably install it just fine.
One of these days, I'll update the conda recipe...
